server/util/index.ts

```diff
@@ -0,0 +1,14 @@
+import { parseAddress } from 'addresser';
+
+export function createKeys(property: string, values: string[]) {
+  return values.map((v) => ({ [property]: v }));
+}
+
+export function formatAddress(address: string): string {
+  let addressString = address;
+  if (!address.includes(',')) {
+    addressString += ', Ithaca, NY 14850';
```

Comment:
This would work in most cases, but what if they enter a custom location that is on campus (zip code is different) or not in Ithaca?

Reply:
i think its expected that all locations on campus are provided for, and this would be just for off campus addresses. ill ask to clarify though